lang/lua-struct/Makefile

```diff
@@ -0,0 +1,39 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lua-struct
+PKG_VERSION:=0.9.2
+PKG_RELEASE:=1
+
+PKG_LICENSE:=MIT/X11
+PKG_URL:=https://github.com/iryont/lua-struct
+
+PKG_MAINTAINER:=jasle <jasle@riseup.net>
+
+PKG_VERSION_TAG:=$(PKG_VERSION)-1
```

Comment:
Why do you need to introduce a new variable? No.

Reply:
The version `0.9.2` has a little bit wired version tag `0.9.2-1`. I wanted to avoid adding the suffix in multiple places, so I put the correct tag into a variable.

If it is preferred to at it at multiple places, i can change the Makefile.

Reply:
I stick with having the tag that is different from the version in a variable. We have variables, so why not use them and repeat things?

Reply:
Please dont mark this conversation as resolved since it is not resolved.

Reply:
So... when it has wired version tag, why dont put it exactly to the PKG_VERSION instead of doing new variable? Its just guess, it should work, because basically you are doing the same thing with the PKG_VERSION_TAG. Still my reaosn is that we dont need to have a new variable.

Reply:
Its version `0.9.2` but the tag is `0.9.2-1`. No idea why, but that’s the way it is.